examples/tracking_merlion.py

```diff
@@ -29,6 +29,7 @@
 """
 
 import os
+from datetime import timedelta
```

Comment:
It is similar like `import pandas as pd`.
```suggestion
import datetime as dt
```